cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh

```diff
@@ -13,12 +13,14 @@ CLUSTER_DNS_DOMAIN={{ .Cluster.Spec.ClusterNetwork.ServiceDomain }}
 POD_CIDR={{ .PodCIDR }}
 SERVICE_CIDR={{ .ServiceCIDR }}
 ARCH=amd64
+DISTRIBUTOR=$(lsb_release -cs)
+
 swapoff -a
 # disable swap in fstab
 sed -i.bak -r 's/(.+ swap .+)/#\1/' /etc/fstab
 curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
 touch /etc/apt/sources.list.d/kubernetes.list
-sh -c 'echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list'
+sh -c 'echo "deb http://apt.kubernetes.io/ kubernetes-${DISTRIBUTOR} main" > /etc/apt/sources.list.d/kubernetes.list'
```

Comment:
Ubuntu 18.04 lsb_release -cs returns bionic, but there is no kubernetes-bionic: https://packages.cloud.google.com/apt/dists